netbox/utilities/tests/test_filters.py

```diff
@@ -60,3 +65,196 @@ def test_filter_combined(self):
         self.assertEqual(qs.count(), 2)
         self.assertEqual(qs[0], self.site1)
         self.assertEqual(qs[1], self.site3)
+
+
+class DynamicFilterLookupExpressionTest(TestCase):
```

Comment:
IMO we should also include a generic test for BaseFilterSet itself. Just an instantiation with one each of every filter type, with tests to ensure all the expected filters are present. I'm going to take a shot at this myself (unless there are objections) to help get more familiar with the logic.

Reply:
Yeah sounds good. I will hold off doing it unless you ask me.